IKZ_plugin/src/ikz_plugin/utils.py

```diff
@@ -102,3 +103,52 @@ def create_archive(
     return get_reference(
         context.upload_id, get_entry_id_from_file_name(filename, context.upload_id)
     )
+
+
+def create_timeseries_objects(
```

Comment:
add docstring

Reply:
I moved this function into movpe1 folder